app/dashboard/src/layouts/AssetsTable.tsx

```diff
@@ -999,8 +999,7 @@ export default function AssetsTable(props: AssetsTableProps) {
     ],
     { queryKey: [], staleTime: 0, meta: { persist: false } }
   )
-  const isLoading =
-    rootDirectoryQuery.isLoading || rootDirectoryQuery.isPending || rootDirectoryQuery.isFetching
+  const isLoading = rootDirectoryQuery.isLoading || rootDirectoryQuery.isPending
```

Comment:
I think `isLoading` should be enough.
 
![CleanShot 2024-07-26 at 12 03 01@2x](https://github.com/user-attachments/assets/1daa6f9f-8d3e-418a-a807-b2120894f37d)